webcompat/static/js/lib/issues.js

```diff
@@ -530,6 +530,7 @@ issues.MainView = Backbone.View.extend(
           commentLinkId: null
         });
         this.addComment(newComment);
+        newComment.attributes.body = textarea.val();
```

Comment:
What about doing the opposite? The only reason we have html is because of this.

```js
 body: this.sanitizeMarkdown(md.render(textarea.val())),
```

Why not changing the rendering of the text in `addComment` instead of `addNewComment`, and just use the text version in `addNewComment`



So we really separate what is done for posting the comment to GitHub and what is done for displaying the comment.

Reply:
ah probably not working because addComment is used in other contexts which are not the new comment and where the comment is already HTML.

ok let's see.

Reply:
```js
    addNewComment: function(event) {
      var form = $(".js-Comment-form");
      var textarea = $(".js-Comment-text");

      if (form[0].checkValidity()) {
        event.preventDefault();
        var newComment = new issues.Comment({
          avatarUrl: form.data("avatarUrl"),
          body: textarea.val(),
          commenter: form.data("username"),
          createdAt: moment(new Date().toISOString()).fromNow(),
          commentLinkId: null
        });
        // Displays the comment for the user after formatting it as HTML
        var displayComment = newComment;
        displayComment.attributes.body = this.sanitizeMarkdown(md.render(textarea.val()));
        this.addComment(displayComment);
        // Push the markdown text comment to GitHub
        newComment.save();
      }
},
```

Do you think this would work?

Reply:
I've just tested it and it doesn't. It's sending the comments as HTML.
Which is **really** weird, I don't know why it's doing it. Your code should work...

Reply:
Humm... maybe when you call `addComment` you're overwriting the real value of newComment for the displayComment value?

Reply:
I printed the value of the comment inside `addComment` and before `newComment.save()` and I think that's what's happening.

Reply:
ah yes my bad because the assignment is not creating a new object but just a pointer.
We would need to clone it. ok sorry for this.

Reply:
You think we need to change something? @karlcow

Reply:
nope let's go with what you did. Thanks a lot.